package.json

```diff
@@ -15,7 +15,7 @@
         "@angular/core": "6.1.2",
         "@bazel/benchmark-runner": "0.1.0",
         "@bazel/ibazel": "0.4.0",
-        "@bazel/typescript": "0.16.0",
+        "@bazel/typescript": "github:alexeagle/rules_typescript#e78d61dba94a1c5fc1e7f0deb5c8058b04a7bbfb",
```

Comment:
Needs to be changed after https://github.com/bazelbuild/rules_typescript/pull/256 is merged and released.